.github/workflows/test.yml

```diff
@@ -316,13 +322,19 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
+        engine: ['napi', 'binary']
         os: [ubuntu-20.04] # macos-latest, windows-latest
         node: [12]
 
     steps:
       - uses: actions/checkout@v2
 
       - run: docker-compose -f src/docker/docker-compose.yml up --detach postgres mysql mssql
+      
+      - name: Set Engine Type
+        if: ${{ matrix.engine == 'napi' }}
+        run: |
+          echo "PRISMA_FORCE_NAPI=true" >> $GITHUB_ENV
```

Comment:
Why is napi needed for migrate?

Reply:
Migrate use the engine-commands `getConfig` and `getDMMF`? This is ensuring that the NAPI implementation of these commands does not breaking anything in migrate

Reply:
Sure!